common-files/Jenkinsfile

```diff
@@ -5,7 +5,6 @@
 buildPlugin(
   useContainerAgent: true,
   configurations: [
-    [platform: 'linux', jdk: 11], // use 'docker' if you have containerized tests
-    [platform: 'windows', jdk: 11],
     [platform: 'linux', jdk: 17]
```

Comment:
```suggestion
    [platform: 'linux', jdk: 17], // use 'docker' if you have containerized tests
```

Reply:
this isn't very useful it doesn't tell you how to use docker but sure if it preserves existing =/